api/graphql/base_mutations.py

```diff
@@ -8,3 +10,17 @@ def mutate_and_get_payload(cls, root, info, **input):
             return None
 
         return super().mutate_and_get_payload(root, info, **input)
+
+
+class AuthDeleteMutation(AuthMutation):
+    deleted = graphene.Boolean()
+
+    class Input:
+        pk = graphene.ID()
+
+    @classmethod
+    def mutate_and_get_payload(cls, root, info, **input):
+        if cls.has_permission(root, info, input):
+            space = get_object_or_404(cls.model, pk=input["pk"])
```

Comment:
I guess this could have more generic name than space.

Reply:
indeed :+1: